docker-for-mac/release-notes.md

```diff
@@ -526,6 +526,16 @@ events or unexpected unmounts.
 
 ## Edge Release Notes
 
+### Docker Community Edition 18.03.0-ce-rc4-mac57 2018-03-15 (Edge)
+
+* Upgrades
+  - [Docker 18.03.0-ce-rc4](https://github.com/docker/docker-ce/releases/tag/v18.03.0-ce-rc4)
+  - AUFS 20180312
+
+* Bug fixes and minor changes
+  - Fix support for AUFS. Fixes [docker/for-win#1831](https://github.com/docker/for-win/issues/1831)
```

Comment:
@gtardif ... is this right? This is a D4W fix.

Reply:
_The issue affects both mac & windows. The for-win reference might help users to understand what this is about more in detail, even for mac users._